src/datetime/mod.rs

```diff
@@ -1878,6 +1881,8 @@ where
     }
 }
 
+const UNIX_EPOCH_DAY: i64 = 719_163;
```

Comment:
Pre-existing issue, but can you add a comment on what this number is/how you can check it?

Reply:
Like I know 😆. Added a comment.